README.md

```diff
@@ -334,5 +334,24 @@ we have put the relevant configuration in the `exampleConfig` property within th
 Note that the format and contents of this document (at least the type-specific part) are to be understood by the extension source implementation.
 Specifically, this JSON document is the configuration document delivered as part of the [`configureExtension`](#configureExtension_operation) operation.
 
+# Developer Notes
+To develop or build within this environment, we use `gradle` to build.
+
+Some of the connectors require other software to build.
+Generally, these are things that are not available a maven downloads,
```

Comment:
"are not available a maven downloads"

Should this say "as maven downloads"?

Reply:
Indeed.  Thanks.  Cleaned up a few other wording things as well.